sample.env

```diff
@@ -0,0 +1,5 @@
+POSTGRES_DB=postgres
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=adminpwd
+PGADMIN_DEFAULT_EMAIL=xx@xx.uu.com
+PGADMIN_DEFAULT_PASSWORD=defaultuserpwd
```

Comment:
I would have to add later SERVERNAME=localhost, for telling COCO if the code is being run locally or in the server.

Reply:
It is not that I need to acquire that server runs on localhost, but a variable like APP_ENV=server or APP_ENV=local